MSAL/src/MSALPublicClientApplication.m

```diff
@@ -1033,10 +1033,11 @@ - (BOOL)removeAccount:(MSALAccount *)account
         }
     }
 
-    if (self.accountMetadataCache && ![self.accountMetadataCache clearForHomeAccountId:account.identifier
-                                                                              clientId:self.internalConfig.clientId
-                                                                               context:nil
-                                                                                 error:error])
+    if (self.accountMetadataCache && ![self.accountMetadataCache updateSignInStateForHomeAccountId:account.identifier
```

Comment:
should we get rid of removeAdditionalAccountInfo?

Reply:
I reverted that change in this PR, because I want to put those changes into a separate PR "MSAL integrated with accountMetadata for sign out state" :)

Reply:
Ok, makes sense :)